uapi/uapi/svc_exchange.rs

```diff
@@ -3,7 +3,9 @@
 
 use crate::systypes::{EraseMode, EraseType, Status};
 
+/// SVC Exchange area len. TODO: to be kconfig-generated
 pub const SVC_EXCH_AREA_LEN: usize = 128; // TODO: replace by CONFIG-defined value
```

Comment:
Is this still used ?

Reply:
By now yes, but by nobody.... (extern C do not use this, shield-rs doesn't exist. This requires to be fixed though, but in another PR. Checking for issue